packages/notifi-react-hooks/lib/hooks/useNotifiClient.ts

```diff
@@ -247,6 +248,10 @@ const useNotifiClient = (
   const [isAuthenticated, setIsAuthenticated] = useState<boolean>(false);
   const [expiry, setExpiry] = useState<string | null>(null);
   const [isTokenExpired, setisTokenExpired] = useState<boolean>(false);
+
+  const reenableSmsPhoneNumber =
+    env === 'Production' ? '+1 206 222 3465' : '+1 253 880 1477 ';
```

Comment:
do we have to expose this from the client? why not have these two values in the same file as the error?